modules/schema/src/test/java/org/apache/ignite/internal/schema/SchemaManagerTest.java

```diff
@@ -72,14 +71,12 @@ class SchemaManagerTest extends BaseIgniteAbstractTest {
 
     private static final long CAUSALITY_TOKEN_1 = 0;
     private static final long CAUSALITY_TOKEN_2 = 45;
+    private static final long CAUSALITY_TOKEN_3 = 56;
```

Comment:
Because in `SchemaManagerTest#create2TableVersions` we complete with the first two tokens and the third is needed to complete with it below in the test.

Reply:
But what changed in the test that you needed to change this?

Reply:
In the test, the version value was completed with the same token twice, which is an error.